sql/opt/exec/execbuilder: TestExecBuild failed

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2195092&tab=buildLog) on [master@2dd0592027ed659d4c461359cd7933b0021a0f02](https://github.com/cockroachdb/cockroach/commits/2dd0592027ed659d4c461359cd7933b0021a0f02):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (79.27s)
    logic.go:2785: randomize batchSize to 1
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestExecBuild060739204
    test_log_scope.go:63: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/inverted_index
=== PAUSE TestExecBuild/local/inverted_index
=== CONT  TestExecBuild/local/inverted_index
--- done: testdata/ordinality with config local: 6 tests, 0 failures
--- progress: testdata/secondary_index_column_families: 34 statements/queries
        --- FAIL: TestExecBuild/local/inverted_index (2.85s)
            logic.go:2075: 
                 
                testdata/inverted_index:164: EXPLAIN (VERBOSE) SELECT * from d where b @>'{"a": "b"}'
                expected:
                    ·           distribution         local                        ·       ·
                    ·           vectorized           true                         ·       ·
                    index join  ·                    ·                            (a, b)  ·
                     │          estimated row count  111 (missing stats)          ·       ·
                     │          table                d@primary                    ·       ·
                     │          key columns          a                            ·       ·
                     └── scan   ·                    ·                            (a)     ·
                    ·           estimated row count  111 (missing stats)          ·       ·
                    ·           table                d@foo_inv                    ·       ·
                    ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
                    
                but found (query options: "") :
                    ·           distribution         local                        ·       ·
                    ·           vectorized           true                         ·       ·
                    index join  ·                    ·                            (a, b)  ·
                     │          estimated row count  111 (missing stats)          ·       ·
                     │          table                d@primary                    ·       ·
                     │          key columns          a                            ·       ·
                     └── scan   ·                    ·                            (a)     ·
                    ·           estimated row count  110 (missing stats)          ·       ·
                    ·           table                d@foo_inv                    ·       ·
                    ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
                    
            logic.go:2656: 
                testdata/inverted_index:178: error while processing
            logic.go:2656: testdata/inverted_index:178: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2195190&tab=buildLog) on [master@e19eec29c27d8436f5cc5a1ea399493fb719e8b7](https://github.com/cockroachdb/cockroach/commits/e19eec29c27d8436f5cc5a1ea399493fb719e8b7):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (76.69s)
    logic.go:2785: randomize batchSize to 3
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestExecBuild230200129
    test_log_scope.go:63: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/inverted_index
=== PAUSE TestExecBuild/local/inverted_index
=== CONT  TestExecBuild/local/inverted_index
        --- FAIL: TestExecBuild/local/inverted_index (2.05s)
            logic.go:2075: 
                 
                testdata/inverted_index:164: EXPLAIN (VERBOSE) SELECT * from d where b @>'{"a": "b"}'
                expected:
                    ·           distribution         local                        ·       ·
                    ·           vectorized           true                         ·       ·
                    index join  ·                    ·                            (a, b)  ·
                     │          estimated row count  111 (missing stats)          ·       ·
                     │          table                d@primary                    ·       ·
                     │          key columns          a                            ·       ·
                     └── scan   ·                    ·                            (a)     ·
                    ·           estimated row count  111 (missing stats)          ·       ·
                    ·           table                d@foo_inv                    ·       ·
                    ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
                    
                but found (query options: "") :
                    ·           distribution         local                        ·       ·
                    ·           vectorized           true                         ·       ·
                    index join  ·                    ·                            (a, b)  ·
                     │          estimated row count  111 (missing stats)          ·       ·
                     │          table                d@primary                    ·       ·
                     │          key columns          a                            ·       ·
                     └── scan   ·                    ·                            (a)     ·
                    ·           estimated row count  110 (missing stats)          ·       ·
                    ·           table                d@foo_inv                    ·       ·
                    ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
                    
            logic.go:2656: 
                testdata/inverted_index:178: error while processing
            logic.go:2656: testdata/inverted_index:178: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2195223&tab=buildLog) on [master@34bdde523b5717eb612893566bd86764bbae1387](https://github.com/cockroachdb/cockroach/commits/34bdde523b5717eb612893566bd86764bbae1387):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (75.98s)
    logic.go:2785: randomize batchSize to 1
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestExecBuild990061069
    test_log_scope.go:63: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/inverted_index
=== PAUSE TestExecBuild/local/inverted_index
=== CONT  TestExecBuild/local/inverted_index
--- done: testdata/explain with config local: 90 tests, 0 failures
        --- FAIL: TestExecBuild/local/inverted_index (2.25s)
            logic.go:2075: 
                 
                testdata/inverted_index:164: EXPLAIN (VERBOSE) SELECT * from d where b @>'{"a": "b"}'
                expected:
                    ·           distribution         local                        ·       ·
                    ·           vectorized           true                         ·       ·
                    index join  ·                    ·                            (a, b)  ·
                     │          estimated row count  111 (missing stats)          ·       ·
                     │          table                d@primary                    ·       ·
                     │          key columns          a                            ·       ·
                     └── scan   ·                    ·                            (a)     ·
                    ·           estimated row count  111 (missing stats)          ·       ·
                    ·           table                d@foo_inv                    ·       ·
                    ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
                    
                but found (query options: "") :
                    ·           distribution         local                        ·       ·
                    ·           vectorized           true                         ·       ·
                    index join  ·                    ·                            (a, b)  ·
                     │          estimated row count  111 (missing stats)          ·       ·
                     │          table                d@primary                    ·       ·
                     │          key columns          a                            ·       ·
                     └── scan   ·                    ·                            (a)     ·
                    ·           estimated row count  110 (missing stats)          ·       ·
                    ·           table                d@foo_inv                    ·       ·
                    ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
                    
            logic.go:2656: 
                testdata/inverted_index:178: error while processing
            logic.go:2656: testdata/inverted_index:178: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2195250&tab=buildLog) on [master@2dd0592027ed659d4c461359cd7933b0021a0f02](https://github.com/cockroachdb/cockroach/commits/2dd0592027ed659d4c461359cd7933b0021a0f02):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (66.99s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestExecBuild594816714
    test_log_scope.go:63: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/inverted_index
=== PAUSE TestExecBuild/local/inverted_index
=== CONT  TestExecBuild/local/inverted_index
        --- FAIL: TestExecBuild/local/inverted_index (2.94s)
            logic.go:2075: 
                 
                testdata/inverted_index:164: EXPLAIN (VERBOSE) SELECT * from d where b @>'{"a": "b"}'
                expected:
                    ·           distribution         local                        ·       ·
                    ·           vectorized           true                         ·       ·
                    index join  ·                    ·                            (a, b)  ·
                     │          estimated row count  111 (missing stats)          ·       ·
                     │          table                d@primary                    ·       ·
                     │          key columns          a                            ·       ·
                     └── scan   ·                    ·                            (a)     ·
                    ·           estimated row count  111 (missing stats)          ·       ·
                    ·           table                d@foo_inv                    ·       ·
                    ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
                    
                but found (query options: "") :
                    ·           distribution         local                        ·       ·
                    ·           vectorized           true                         ·       ·
                    index join  ·                    ·                            (a, b)  ·
                     │          estimated row count  111 (missing stats)          ·       ·
                     │          table                d@primary                    ·       ·
                     │          key columns          a                            ·       ·
                     └── scan   ·                    ·                            (a)     ·
                    ·           estimated row count  110 (missing stats)          ·       ·
                    ·           table                d@foo_inv                    ·       ·
                    ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
                    
            logic.go:2656: 
                testdata/inverted_index:178: error while processing
            logic.go:2656: testdata/inverted_index:178: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/sql/opt/exec/execbuilder/testdata/inverted_index
+++ b/pkg/sql/opt/exec/execbuilder/testdata/inverted_index
@@ -171,7 +171,7 @@ index join  ·                    ·                            (a, b)  ·
  │          table                d@primary                    ·       ·
  │          key columns          a                            ·       ·
  └── scan   ·                    ·                            (a)     ·
-·           estimated row count  111 (missing stats)          ·       ·
+·           estimated row count  110 (missing stats)          ·       ·
 ·           table                d@foo_inv                    ·       ·
 ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
 
@@ -185,7 +185,7 @@ index join  ·                    ·                                        (a, 
  │          table                d@primary                                ·       ·
  │          key columns          a                                        ·       ·
  └── scan   ·                    ·                                        (a)     ·
-·           estimated row count  111 (missing stats)                      ·       ·
+·           estimated row count  110 (missing stats)                      ·       ·
 ·           table                d@foo_inv                                ·       ·
 ·           spans                /"a"/"b"/Arr/1-/"a"/"b"/Arr/1/PrefixEnd  ·       ·
 
@@ -199,7 +199,7 @@ index join  ·                    ·                                            
  │          table                d@primary                                        ·       ·
  │          key columns          a                                                ·       ·
  └── scan   ·                    ·                                                (a)     ·
-·           estimated row count  111 (missing stats)                              ·       ·
+·           estimated row count  110 (missing stats)                              ·       ·
 ·           table                d@foo_inv                                        ·       ·
 ·           spans                /"a"/"b"/Arr/Arr/2-/"a"/"b"/Arr/Arr/2/PrefixEnd  ·       ·
 
@@ -213,7 +213,7 @@ index join  ·                    ·                             (a, b)  ·
  │          table                d@primary                     ·       ·
  │          key columns          a                             ·       ·
  └── scan   ·                    ·                             (a)     ·
-·           estimated row count  111 (missing stats)           ·       ·
+·           estimated row count  110 (missing stats)           ·       ·
 ·           table                d@foo_inv                     ·       ·
 ·           spans                /"a"/"b"/True-/"a"/"b"/False  ·       ·
 
@@ -227,7 +227,7 @@ index join  ·                    ·                        (a, b)  ·
  │          table                d@primary                ·       ·
  │          key columns          a                        ·       ·
  └── scan   ·                    ·                        (a)     ·
-·           estimated row count  111 (missing stats)      ·       ·
+·           estimated row count  110 (missing stats)      ·       ·
 ·           table                d@foo_inv                ·       ·
 ·           spans                /Arr/1-/Arr/1/PrefixEnd  ·       ·
 
@@ -241,7 +241,7 @@ index join  ·                    ·                                            
  │          table                d@primary                                        ·       ·
  │          key columns          a                                                ·       ·
  └── scan   ·                    ·                                                (a)     ·
-·           estimated row count  111 (missing stats)                              ·       ·
+·           estimated row count  110 (missing stats)                              ·       ·
 ·           table                d@foo_inv                                        ·       ·
 ·           spans                /Arr/"a"/"b"/Arr/1-/Arr/"a"/"b"/Arr/1/PrefixEnd  ·       ·
 
@@ -283,7 +283,7 @@ index join  ·                    ·                            (a, b)  ·
  │          table                d@primary                    ·       ·
  │          key columns          a                            ·       ·
  └── scan   ·                    ·                            (a)     ·
-·           estimated row count  111 (missing stats)          ·       ·
+·           estimated row count  110 (missing stats)          ·       ·
 ·           table                d@foo_inv                    ·       ·
 ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
 
@@ -297,7 +297,7 @@ index join  ·                    ·                                    (a, b)  
  │          table                d@primary                            ·       ·
  │          key columns          a                                    ·       ·
  └── scan   ·                    ·                                    (a)     ·
-·           estimated row count  111 (missing stats)                  ·       ·
+·           estimated row count  110 (missing stats)                  ·       ·
 ·           table                d@foo_inv                            ·       ·
 ·           spans                /"a"/"c"/"b"-/"a"/"c"/"b"/PrefixEnd  ·       ·
 
@@ -318,7 +318,7 @@ index join  ·                    ·                            (a, b)  ·
  │          table                d@primary                    ·       ·
  │          key columns          a                            ·       ·
  └── scan   ·                    ·                            (a)     ·
-·           estimated row count  111 (missing stats)          ·       ·
+·           estimated row count  110 (missing stats)          ·       ·
 ·           table                d@foo_inv                    ·       ·
 ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
 
@@ -512,11 +512,11 @@ filter           ·                    ·                         (a, b)  ·
  │               estimated row count  12 (missing stats)        ·       ·
  │               filter               b @> '{"a": {}, "b": 2}'  ·       ·
  └── index join  ·                    ·                         (a, b)  ·
-      │          estimated row count  111 (missing stats)       ·       ·
+      │          estimated row count  110 (missing stats)       ·       ·
       │          table                d@primary                 ·       ·
       │          key columns          a                         ·       ·
       └── scan   ·                    ·                         (a)     ·
-·                estimated row count  111 (missing stats)       ·       ·
+·                estimated row count  110 (missing stats)       ·       ·
 ·                table                d@foo_inv                 ·       ·
 ·                spans                /"b"/2-/"b"/2/PrefixEnd   ·       ·
 
@@ -546,7 +546,7 @@ index join  ·                    ·                    (a, b)  ·
  │          table                e@primary            ·       ·
  │          key columns          a                    ·       ·
  └── scan   ·                    ·                    (a)     ·
-·           estimated row count  111 (missing stats)  ·       ·
+·           estimated row count  110 (missing stats)  ·       ·
 ·           table                e@e_b_idx            ·       ·
 ·           spans                /1-/2                ·       ·
 

